pages/index.js

```diff
@@ -2,7 +2,7 @@ import Head from "next/head";
 import Content from "../components/content";
 import Header from "./components/Header";
 import Hero from "../components/hero/Hero";
-
+import Footer from "./components/Footer";
 export default function Home() {
```

Comment:
Please add space between imports and function for better readability and code consistency.

Reply:
How does that look @YuriDevAT?

Reply:
```
import Content from "../components/content";
import Header from "./components/Header";
import Hero from "../components/hero/Hero";
import Footer from "./components/Footer";

export default function Home() {
...
}
```

Reply:
Got it!